### ISSUE TITLE
nodejs needs to be running locally for this?

### ISSUE THREAD

**Issue by dakipro** [zoton2/node-streamdeck-util#8]
sorry if it is stupid or obvious question, but NodeJs needs to be running independently of the plugin for this to work?
Or can plugin somehow get the NodeJS to run when it is loaded or... something? :)

**Comment by zoton2** [comment]
Yes, it is designed for it to be run independently and they just communicate via WebSockets.

If you just want the Stream Deck software to run a Node.js script for you, I think that's possible although I've not done it myself. It shouldn't need this library, but I'm not sure how to do it off the top of my head. Maybe there's some stuff on Google; [this reddit post](https://www.reddit.com/r/StreamDeckSDK/comments/aiuho3/introducing_nodejs_base_for_plugins/) from a few years ago was something I saw back then but their repository seems to have now been deleted.
